lib/txgh.rb

```diff
@@ -0,0 +1,12 @@
+module Txgh
+  autoload :Application,      'txgh/app'
```

Comment:
assuming this is activesupport's autoload, would autoload_at be easier and just pass the path?